test/e2e/federation-replicaset.go

```diff
@@ -92,21 +92,22 @@ var _ = framework.KubeDescribe("Federation replicasets [Feature:Federation]", fu
 		})
 
 		It("should create and update matching replicasets in underling clusters", func() {
-			rs := createReplicaSetOrFail(f.FederationClientset_1_4, f.Namespace.Namespace)
+			nsName := f.FederationNamespace.Name
+			rs := createReplicaSetOrFail(f.FederationClientset_1_4, nsName)
 			defer func() { // cleanup. deletion of replicasets is not supported for underling clusters
-				By(fmt.Sprintf("zero replicas then delete replicaset %q/%q", f.Namespace.Name, rs.Name))
+				By(fmt.Sprintf("zero replicas then delete replicaset %q/%q", nsName, rs.Name))
```

Comment:
nit: You could replace nsName + rs.Name with types.NamespacedName{... } which has a String() method.